src/Tools/dotnet-monitor/ParameterCapturing/ParameterCapturingEvents.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.Diagnostics.Tools.Monitor.ParameterCapturing
+{
+    internal static class ParameterCapturingEvents
+    {
+        public const string SourceName = "Microsoft.Diagnostics.Monitoring.ParameterCapturing";
+
+        public static class EventIds
+        {
+            public const int Flush = 1;
+
+            public const int CapturingStart = 2;
+            public const int CapturingStop = 3;
+            public const int FailedToCapture = 4;
+            public const int ServiceNotAvailable = 5;
+        }
+
+
+        public enum ServiceNotAvailableReason : uint
```

Comment:
Maybe `ServiceStatus` with `NotStarted = 0`, `Running`, `NotSupported`, and `GenericError` states? This would eliminate the need to make this any usage of this nullable.